src/components/v2/Table/index.stories.tsx

```diff
@@ -113,3 +118,19 @@ export const WithCustomColumnsWidth = () => (
     gridTemplateColumns="100px 1fr 1fr 140px"
   />
 );
+
+export const MobileView = () => (
+  <Table columns={columns} data={rows} title="Market Data" rowKeyIndex={0} isMobileView />
+);
+
+export const MobileViewMultipleRows = () => (
+  <Table
+    columns={columns}
+    data={rows}
+    title="Market Data"
+    rowKeyIndex={0}
+    gridTemplateColumns="1fr 150px"
+    gridTemplateRowsMobile="1fr 1fr"
+    isMobileView
```

Comment:
We take snapshots for a variety of view port sizes so we don't need a specific story to show responsiveness. Stories represent different states of the components

Reply:
Agreed, removed